Scenes/Screens/RevivalShop/RevivalShop.gd

```diff
@@ -16,194 +16,112 @@ const BASIC_DAMAGE_PRCE = 0
 const BASIC_HP_PRCE = 0
 const BASIC_ITEMS_PRCE = 0
 
-
-onready var enemies_per_level_count = find_node("Item_enemies_per_level").find_node("Count")
-onready var earn_gold_count = find_node("Item_earn_gold").find_node("Count")
-onready var earn_xp_count = find_node("Item_earn_xp").find_node("Count")
-onready var time_to_kill_boss_count = find_node("Item_time_to_kill_boss").find_node("Count")
-onready var silver_moon_probability_count = find_node("Item_silver_moon_probability").find_node("Count")
-onready var basic_start_level_count = find_node("Item_basic_start_level").find_node("Count")
-onready var basic_damage_count = find_node("Item_basic_damage").find_node("Count")
-onready var basic_hp_count = find_node("Item_basic_hp").find_node("Count")
-onready var items_price_count = find_node("Item_items_price").find_node("Count")
-
-onready var enemies_per_level_button = find_node("Item_enemies_per_level").find_node("Button")
-onready var earn_gold_button = find_node("Item_earn_gold").find_node("Button")
-onready var earn_xp_button = find_node("Item_earn_xp").find_node("Button")
-onready var time_to_kill_boss_button = find_node("Item_time_to_kill_boss").find_node("Button")
-onready var silver_moon_probability_button = find_node("Item_silver_moon_probability").find_node("Button")
-onready var basic_start_level_button = find_node("Item_basic_start_level").find_node("Button")
-onready var basic_damage_button = find_node("Item_basic_damage").find_node("Button")
-onready var basic_hp_button = find_node("Item_basic_hp").find_node("Button")
-onready var items_price_button = find_node("Item_items_price").find_node("Button")
-
 onready var level_manager = get_parent().find_node("LevelManager")
 
 func _ready():
-	connect_buttons_signals()
-
-func _process(delta):
-	set_enemies_per_level_button()
-	set_enemies_per_level_count()
-	set_earn_gold_button()
-	set_earn_gold_count()
-	set_earn_xp_button()
-	set_earn_xp_count()
-	set_time_to_kill_boss_button()
-	set_time_to_kill_boss_count()
-	set_silver_moon_probability_button()
-	set_silver_moon_probability_count()
-	set_basic_start_level_button()
-	set_basic_start_level_count()
-	set_basic_damage_button()
-	set_basic_damage_count()
-	set_basic_hp_button()
-	set_basic_hp_count()
-	set_items_price_button()
-	set_items_price_count()
+	#connect_buttons_signals()
+	send_count_text("Item_enemies_per_level", get_parent().find_node("LevelManager").dwarves_per_level)
+	send_count_text("Item_earn_gold", 'x' + String(GameData.additional_gold_multipler))
+	send_count_text("Item_earn_xp", 'x' + String(GameData.additional_xp_multipler))
```

Comment:
Now in game we don't have xp. You can hide the option Item_earn_xp in RevivalShop.tscn and if you want comment off these type lines.